Automattic/wp-calypso
# Reader: add follow source for site results in search

Part of https://github.com/Automattic/wp-calypso/issues/15060#issuecomment-309598127. Adds a missing `followSource` for site results in search.

### To test

On http://calypso.localhost:3000/read/search?q=banana, ensure that follow buttons in the Sites results are being handed a `followSource` prop.
